Foundation/GTMSignalHandler.h

```diff
@@ -53,16 +53,15 @@
 //
 // Multiple handlers for the same signal is NOT supported.
 //
-// kqueue() is used to handle the signals, and the default runloop for the first
-// signal handler is used for signal delivery, so keep that in mind when you're
-// using this class.  This class explicitly does not handle arbitrary runloops
-// and threading.
-//
+// libdispatch is used to handle the signals. NZB that sending a signal
+// from dispatch_get_main_queue will not get caught in a CFRunLoop based run
+// loop.
+// https://openradar.appspot.com/radar?id=5030997057863680
+
+NS_DEPRECATED(10_0, 10_6, 1_0, 4_0, "Use signal dispatch sources")
```

Comment:
Maybe use an explicit api or a docs pointer for them?

Reply:
Done